x/blocksdk/types/params.go

```diff
@@ -0,0 +1,26 @@
+package types
+
+const (
+	DefaultEnabled = false
+)
+
+// NewParams returns a new Params instance with the provided values.
+func NewParams(
+	enabled bool,
+) Params {
+	return Params{
+		Enabled: enabled,
```

Comment:
Why did we choose to add `Enabled` here?

Reply:
We use this to only use the on-chain lane info if `Enabled` is true.  Otherwise, the Registry works in memory like it used to

Reply:
Is this backwards compatible tho? Like on versions where x/blocksdk is not used, the get would err, no?

Reply:
It's probs nbd, j wanted to check